src/vanilla.ts

```diff
@@ -3,7 +3,6 @@ import { getUntracked, markToTrack } from 'proxy-compare'
 const VERSION = __DEV__ ? Symbol('VERSION') : Symbol()
 const LISTENERS = __DEV__ ? Symbol('LISTENERS') : Symbol()
 const SNAPSHOT = __DEV__ ? Symbol('SNAPSHOT') : Symbol()
-const HANDLER = __DEV__ ? Symbol('HANDLER') : Symbol()
 const PROMISE_RESULT = __DEV__ ? Symbol('PROMISE_RESULT') : Symbol()
 const PROMISE_ERROR = __DEV__ ? Symbol('PROMISE_ERROR') : Symbol()
```

Comment:
For now, I still want to hide them. I might consider combining into one symbol later (then it can ease customizing.) I first tried in this PR, but it made things complicated.

Reply:
Here’s an example of using one symbol only: https://github.com/immerjs/immer/blob/e28b53f2d12696954b1f0125494498e34d77a3df/src/core/proxy.ts#L55

It uses one symbol to access an object which in valtio’s case can contain the listeners and version.

Reply:
Yep. I wish to combine `SNAPSHOT` too. Removing symbols completely and using WeakMaps might be possible, but maybe performance-wise a little worth. Anyway, let's tackle it later. I would like to know what use cases this don't cover (and then the solution is either expose more or fork.)